src/main/resources/assets/nochatreports/lang/en_us.json

```diff
@@ -92,6 +93,16 @@
 	"option.NoChatReports.showNCRButton.tooltip": "Adds a button that toggles most of the mod's client-sided functionality to multiplayer menu.",
 	"option.NoChatReports.enableMod": "Enable mod",
 	"option.NoChatReports.enableMod.tooltip": "Responsible for enabling most of the mod's client-sided functionality, see option above.",
+	"option.NoChatReports.reconnectAwaitSeconds": "Reconnect await seconds",
+	"option.NoChatReports.reconnectAwaitSeconds.tooltip": "Number of seconds which client should wait before automatic reconnect once server rejects connection without chat signing key, assuming user agreed to enter unsafe server.",
+	"option.NoChatReports.postDisconnectAwaitSeconds": "Post disconnect await seconds",
+	"option.NoChatReports.postDisconnectAwaitSeconds.tooltip": "Similar to §oreconnectAwaitSeconds§r, but countdown starts as soon as player disconnects from the server normally. If this delay is greater than §oreconnectAwaitSeconds§r when they try to re-join the server - then it will apply instead.",
+	"option.NoChatReports.signingCheckDelaySeconds": "Signing check delay seconds",
+	"option.NoChatReports.signingCheckDelaySeconds.tooltip": "Delay for checking whether whitelisted servers still enforce chat signing, in seconds. Setting this to 0 will cause check to be performed on every login attempt, similar to non-whitelisted servers.",
+	"option.NoChatReports.serverSigningChecks": "Server signing checks",
```

Comment:
That doesn't seem to be in the screen per code above, why add the string?

Reply:
Well, not that it interferes with anything. Let it be there in case things change in future.